assets/css/properties_panel/_header.scss

```diff
@@ -80,7 +75,15 @@
   }
 }
 
-.m-properties-panel__header > button {
-  flex: 0 0 auto;
+.m-properties-panel__close-ping {
```

Comment:
Currently, the click target for the x is big, including the padding and taking up most of the right side of the header. I think this wrapper might make the click target much smaller, so it's just the x. If that's the case, is there a way to rearrange things so that the click target stays as large as possible?

Reply:
That's interesting. I can add an `onclick` to the entire div. So you'd be able to click on the entire right side to close panel—does that seem like a good approach?

Reply:
Made that update.

Reply:
That'd make the clicking work, but the highlighting+pointer behaviors of the button wouldn't work unless you're actually over the button element.

Could you instead put padding on both the button and the `last-gps-ping` elements separately, instead of the wrapping div?